### ISSUE TITLE
Are `nth-child(n)` variants possibile?

### ISSUE THREAD

**Issue by mrxluk9** [windicss/windicss#766]
I'm building a datable with divs, I know it is a bad idea, but I'm just experimenting something. Every div is basically a table `td`. Forget about the templating directives. I would like add a `border-b` to all cell but the last nth cells, because they belong to the last row.

```html
 <div class="grid grid-cols-3 w-full overflow-y-scroll">
  {{#each @data as |row|}}
    {{#each this.columns as |column|}}
      <div class="p-4px border-r first:border-l nth-child(-n+3):border-b cell">
        {{get row column.key}}
      </div>
    {{/each}}
  {{/each}}
  </div>
```

In this scenario `-n+3` is manually inserted by me or by the templating engine. Is this possibile?
This cose doesn't show the border bottom.

**Comment by GeForcez** [comment]
I don't think that is possible, since you are working with a templating engine can't you just not add that border when you are at your cells that you don't want to use it ?
